src/resources/shared/openshift/setup_openshift_azure.md

```diff
@@ -0,0 +1,7 @@
+### Prerequisites
+
+Before we begin, the following tools need to be downloaded and added to your `$PATH`:
+
+1. **OpenShift installer**, **pull secret**, and **command line interface**. All can be downloaded from
+   [here](https://cloud.redhat.com/openshift/install/aws/installer-provisioned).
```

Comment:
```suggestion
   [here](https://cloud.redhat.com/openshift/install/aws/installer-provisioned).
```

This is an AWS link, and the link text should be descriptive of the link.

Reply:
Its an AWS link because the installers, along with the explanation, are at that link. Binaries can be downloaded from https://console.redhat.com/openshift/downloads but not the usage explanation. This is the same case of GCP cloud doc as well. https://submariner.io/getting-started/quickstart/openshift/gcp-lb/#prerequisites